hub/src/server/drivers/AzDriver.ts

```diff
@@ -76,7 +76,9 @@ class AzDriver implements DriverModel, DriverModelTestMethods {
       if (error.body && error.body.Code === 'ContainerAlreadyExists') {
         logger.info('Container initialized.')
       } else {
+        /* istanbul ignore next */
```

Comment:
Why add these ignores?

Reply:
Because otherwise there is a net-decrease in coverage percentage and codecov (rightly) fails the build check. 
The alternative is to perform even deeper mocking into 3rd party libs to (very) artificially cause these edge-case error handlers to be reached. Not only is this level of mocking is an anti-pattern, but it will also discourage devs from writing defensive error handling code in general - which is already something we should be trying to move closer towards.

Reply:
okay, sounds reasonable to me